subsystem/src/test/java/org/wildfly/extension/elytron/tls/subsystem/JdkUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * JBoss, Home of Professional Open Source
+ *
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.extension.elytron.tls.subsystem;
+
+/**
+ * @author <a href="mailto:ropalka@redhat.com">Richard Opalka</a>
+ */
+final class JdkUtils {
+
+    private static final String javaSpecVersion = System.getProperty("java.specification.version");
+    private static final String javaVendor = System.getProperty("java.vendor");
+
+    private JdkUtils() {}
+
+    static int getJavaSpecVersion() {
+        if ("1.8".equals(javaSpecVersion)) return 8;
```

Comment:
I think we could look into dropping this class since we will want to support JDK 11 and JDK 17.

Reply:
Is there still a need to specify between the IBM and Oracle JDKs, or could that method be dropped as well?

Reply:
Good question. I think search for references to `isIbmJdk` to see where it's being used. I suspect that might still be needed for tests / implementation specific things where there's differences with IBM.

Reply:
It's only used in TLS test cases, because different providers are used (`com.ibm.*` vs `sun.security.ssl.SunJSSE`), and algorithm names change (`IbmX509` vs `SunX509`)

Reply:
Was able to remove the check for version 8, the method itself is still needed because it's used to differentiate between a different JDK `provider-loader` path ([here](https://github.com/cam-rod/elytron-tls/blob/81ff3e82857ec74355ab4e6cb8813e3a3d8dbc42/subsystem/src/test/resources/org/wildfly/extension/elytron/tls/subsystem/tls-oracle13plus.xml#L4), since it changed in Java 13 both this file and `tls-sun.xml` are still needed).